tools/ovc/openvino/__init__.py

```diff
@@ -53,6 +53,7 @@
 
 from openvino._pyopenvino import RemoteContext
 from openvino._pyopenvino import RemoteTensor
+from openvino._pyopenvino import Op
```

Comment:
why do we need it here?

Reply:
Init files should be aligned. We have cmake-level check for that.